### PR TITLE
Fixing errors at motor and users modules.

### PR DESCRIPTION
**Pull request by 6abef**
Declaring variables of each method, adding docstrings, modifing __init__ & __str__ methods and erasing last @classmethod decorator.

### CODE REVIEW

File: motor.py [hunk 1]
```diff
@@ -50,50 +52,64 @@ def set_status(self, state: int) -> None:
...
+        self.cell3 = Cellstatus()
 
-    def display(self) -> None:
-        print( f"In this row: {self.cell1.get_status()} | {self.cell2.get_status()} | {self.cell3.get_status()}")
+    def __str__(self) -> str:
+        return f"{self.cell1.get_status()} | {self.cell2.get_status()} | {self.cell3.get_status()}"
 
     def set_row_status(self, colomn: int, state: int) -> None:
         """Method to change a given cell in colomn 1-3 for blank(1), circle(2) or cross(3) state"""
         match colomn:
```
Comment (Luis-omega) [inline motor.py:69]: This is still a incomplete case expression, at least add a default with an exception please

File: users.py [hunk 2]
```diff
@@ -1,12 +1,19 @@
 """ 
 This file contains the user class and the userslist
 """
+from typing import TypeAlias  # "from typing_extensions" in Python 3.9 and earlier
 
+UserIdType: TypeAlias = int # declare a type alias that coud be changed easily
```
Comment (Luis-omega) [inline users.py:6]: good use of type alias here!

File: motor.py (hunk 1 above)
Comment (Luis-omega) [reply motor.py:69]: this should be properly fixed in #4  but for now the addition of default case is enough

**Review by Luis-omega — CHANGES_REQUESTED** [review]

**Review by Luis-omega — APPROVED** [review]
Thanks, LGTM

File: motor.py (hunk 1 above)
Comment (6abef) [reply motor.py]: I think I just did it in the last commit (although I’ll check it well in the corresponding issue)

**Comment by 6abef** [comment]
So could I merge them now?

**Comment by Luis-omega** [comment]
yes

**Pull request merged**